gammapy/spectrum/tests/test_models.py

```diff
@@ -221,3 +230,55 @@ def test_pwl_index_2_error():
     eflux, eflux_err = pwl.energy_flux_error(1 * u.TeV, 10 * u.TeV)
     assert_quantity_allclose(eflux, 2.302585E-12 * u.Unit('TeV cm-2 s-1'))
     assert_quantity_allclose(eflux_err, 0.2302585E-12 * u.Unit('TeV cm-2 s-1'))
+
+
+@pytest.mark.xfail
+@requires_dependency('sherpa')
+@requires_data('gammapy-extra')
+def test_spectral_model_absorbed_by_ebl():
+
+    # Observation parameters
+    obs_param = ObservationParameters(alpha=0.2 * u.Unit(''),
+                                      livetime=5. * u.h,
+                                      emin=0.08 * u.TeV,
+                                      emax=12 * u.TeV)
+
+    # Target, PKS 2155-304 from 3FHL
+    name = 'test'
+    absorption = Absorption.read('$GAMMAPY_EXTRA/datasets/ebl/ebl_dominguez11.fits.gz')
```

Comment:
Can you make it convenient to access the "built in" EBL absorption models we provide, via
```
class Absorption:
    @classmethod
    def read_builtin(name):
        """"Read one of the built-in absorption models.
        Parameters
        -----------
        name : {'dominquez11', ...}
```